
test('Hello World!', ()=>{

})

test("Failed test case", ()=>{
    throw new Error()
})
